com.kgurgul.cpuinfo.yml

```diff
@@ -0,0 +1,48 @@
+app-id: com.kgurgul.cpuinfo
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: cpuinfo
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=host
+  - --filesystem=xdg-run/gnupg
+  - --device=all
+  - --socket=gpg-agent
```

Comment:
you don't need this if you have `xdg-run/gnupg`. Choose one.

Reply:
`--socket=gpg-agent` is better since it gets the socket path directly from gpgconf, while the filesystem permission assumes it will always be in `xdg-run/gnupg`.

Reply:
Not needed - removed